src/components/MailboxMessage.vue

```diff
@@ -86,7 +123,7 @@ export default {
 			return list.length > 0
 		},
 		showMessage() {
-			return this.hasMessages && this.$route.name === 'message'
+			return (this.folder.isPriorityInbox === true || this.hasMessages) && this.$route.name === 'message'
```

Comment:
I'm not sure if `isPriorityInbox` can have different values and we want this specifically if it's `true`, but if not we might want to skip `=== true` to unify how truthiness is checked with `this.hasMessage`? 😇